README.md

```diff
@@ -8,45 +8,271 @@ native Bitcoin capabilities of creating multisig transactions.
+#rpcbind=<addr>
+# You must specify rpcuser and rpcpassword to secure the JSON-RPC API
+#rpcuser=<username>
+#rpcpassword=YourSuperGreatPasswordNumber_DO_NOT_USE_THIS_OR_YOU_WILL_GET_ROBBED_385593
+```
+
+These rpc settings will be used by the service.
+
+After creating configuration file, launch `bitcoind` daemon via command:
+
+```shell
+bitcoind --daemon
+```
+
+Downloading and indexing of the bitcoin blockchain may take a lot of time,
+especially for the mainnet.
+
 ## Usage
 
-The anchoring service depends on bitcoind. For the correct work, you need to
-launch bitcoind with specific configuration, see [deployment guide](DEPLOY.md)
-for details.
+Include `exonum-btc-anchoring` as a dependency in your `Cargo.toml`:
+
+```toml
+[dependencies]
+exonum-btc-oracle = "*"
+```
+
+Add the BTC anchoring service to the blockchain in the main project file:
+
+```rust
+extern crate exonum;
+extern crate exonum_btc_anchoring as anchoring;
+
+use exonum::helpers;
+use exonum::helpers::fabric::NodeBuilder;
+
+fn main() {
+    exonum::crypto::init();
+    helpers::init_logger().unwrap();
+    let node = NodeBuilder::new()
+        .with_service(Box::new(anchoring::ServiceFactory));
+    node.run();
+}
+
+```
+
+## Configuration parameters
+
+### For the `generate-template` subcommand
+
+* `btc-anchoring-network` - bitcoin network type used for downloading Bitcoin blocks headers.
+
+  Possible values: [mainnet, testnet, regtest]
+
+* `btc-anchoring-interval` - interval in blocks between anchored blocks.
+* `btc-anchoring-fee` - transaction fee per byte in satoshi that anchoring nodes should use.
+* `btc-anchoring-utxo-confirmations` - the minimum number of confirmations for the first funding transaction.
+
+### For the `generate-config` subcommand
+
+* `btc-anchoring-rpc-host` - Bitcoin rpc url.
+* `btc-anchoring-rpc-user` - User to login into bitcoind.
+* `btc-anchoring-rpc-password` - Password to login into bitcoind.
+
+### For the `finalize` subcommand
+
+* `btc-anchoring-create-funding-tx` - if this option is set, node will create an initial funding
+  transaction with the given amount in satoshis and return it identifier.
+* `btc-anchoring-funding-txid` - Identifier of the initial funding transaction which was created
+  previously using the option above.
+
+### For adjusting the running blockchain configuration
+
+Variables that you can modify
+
+* `transaction_fee` - the amount of the fee per byte in satoshis for the anchoring transactions.
+* `anchoring_interval` - the interval in blocks between anchored blocks.
+* `funding_transaction` - the hex representation of current funding transaction,
+  node would use it as input if it did not spent.
+* `public_keys` - the list of hex-encoded compressed bitcoin public keys of
+  exonum validators that collects into the current anchoring address.
+
+***Warning!** The `network` parameter shouldn't be changed otherwise the service will come to a halt.*
```

Comment:
Not an issue, but his line broke github syntax highlighting. I think you can use `_` instead of `*` here